extensions/blocks/amazon/index.js

```diff
@@ -18,11 +19,15 @@ import { supportsCollections } from '../../shared/block-category';
 
 export const name = 'amazon';
 export const title = __( 'Amazon', 'jetpack' );
+export const iconColor = getIconColor();
```

Comment:
Since we're not using that constant anywhere else in the block files, we probably don't need to export it; you could use `getIconColor()` right away when defining the icon.

Reply:
Good point — will update those.

Reply:
Done in https://github.com/Automattic/jetpack/pull/15311/commits/7779dded8f0131ba5101d0f6c04a1d787712b98e